README.md

```diff
@@ -1 +1,207 @@
-# oci-kubernetes-monitoring
\ No newline at end of file
+# Monitoring Solution for Kubernetes
+
+## About
+
+This provides an end-to-end monitoring solution for Oracle Container Engine for Kubernetes (OKE) and other forms of Kubernetes Clusters, 
+using Logging Analytics, Monitoring other Oracle Cloud Infrastructure (OCI) Services.
```

Comment:
This provides an end-to-end monitoring solution for Oracle Container Engine for Kubernetes (OKE) and other forms of Kubernetes Clusters using Logging Analytics, Monitoring and other Oracle Cloud Infrastructure (OCI) Services.